src/pages/checkout/index.js

```diff
@@ -1,13 +1,59 @@
-import React from 'react';
+import React, { useContext, useState } from 'react';
+import _, { sum } from 'lodash'
```

Comment:
If you're importing _ you don't need to have {sum}

Reply:
I think this may have been auto-filled as I thought that importing '_' just gave you the whole library anyway, ill get rid!

Reply:
Yeah VScode sometimes add things you don't want which can be annoying